settings.gradle

```diff
@@ -1,9 +1,11 @@
 include 'customers-service'
 include 'customers-service-reactive'
 include 'vets-service'
-include 'visits-service'
+//include 'visits-service'
```

Comment:
Have you considered removing visits completely instead of commenting it out since it is no longer used?